gadget/layout.go

```diff
@@ -189,6 +193,69 @@ func LayoutVolumePartially(volume *Volume, constraints LayoutConstraints) (*Part
 	return vol, nil
 }
 
+func resolveOne(gadgetRootDir, kernelRootDir string, kernelInfo *kernel.Info, pathOrRef string) (string, error) {
+	// content may refer to "$kernel:<name>/<content>"
+	if strings.HasPrefix(pathOrRef, "$kernel:") {
```

Comment:
Is this referencing the running kernel or the active kernel or the kernel we are refreshing to? I think this clarification applies not here but to all the places that provide `kernelRootDir`. It should also be documented.

Reply:
It depends on the context If we install a new gadget that has a new "$kernel:name/content" ref then this denotes the running kernel. If we refresh to a new kernel then this refers to the new kernel. I add a comment.